frontend/src/components/pages/AdminDashboard/mockData.js

```diff
@@ -1,406 +0,0 @@
-const data = [
```

Comment:
do we not need this mock data anymore?

Reply:
No the mock data was temporary until our admin dashboard can support data from our endpoints which is being done in this PR